src/views/Give/CausesDashboard.tsx

```diff
@@ -58,6 +58,8 @@ export default function CausesDashboard() {
         provider,
         address,
         networkID: networkId,
+        version2: false,
```

Comment:
@0xLienid Do you know the implications of the version2 and rebase parameters? They were added to IActionValueAsyncThunk, but there's no documentation (groan) in interfaces.ts on what they mean.

Reply:
looks like version2 is just a boolean used for all v2 assets like OHM and sOHM to allow for separate logic and contracts for each. shouldn't be relevant to us so can just leave it as false. rebasing is for a parameter on the new v2 stake function. Again, shouldn't be relevant to us as we aren't swapping in and out of assets or staking/unstaking.